src/index.tsx

```diff
@@ -26,6 +26,9 @@ const theme = createTheme({
 });
 
 const modals = {
+  changeNickname: ContextModal,
+  addCategory: ContextModal,
+  addProduct: ContextModal,
   formModal: ContextModal,
```

Comment:
do these modals differ in appearance from formModal except for content? If not, there is no point in registering so many modals if only the content is different.